### ISSUE TITLE
Template validation added in pr https://github.com/apigee/apigeelint/pull/383 is incorrect

### ISSUE THREAD

**Issue by edekok-mimecast** [apigee/apigeelint#386]
The code dos not seem to work with a JSON template e.g.
```JSON
<Template>
{
   "field": "{some.value}"
}
</Template>
```

Change in `lib/package/plugins/PO026-assignVariableUsage.js` lines 127-130 are causing valid templates to fail
```js
if ( ! pluginUtil.isValidTemplate(templateText)) {
  addMessage(textnode[0].lineNumber, textnode[0].columnNumber,
           "The text of the Template element must be a valid message template");
}
```
due to `lib/package/plugins/_pluginUtil.js: isValidTemplate` not expecting `{` when in `STATES.INSIDE_CURLY:`

**Comment by ssvaidyanathan** [comment]
@DinoChiesa - FYI

**Comment by DinoChiesa** [comment]
Ack.  This is fixed in #385

**Comment by ssvaidyanathan** [comment]
Fixed and released in v2.42.0
Please test and let us know
